addons/website_profile/views/website_profile.xml

```diff
@@ -561,7 +561,7 @@
                 <span class="badge badge-danger font-weight-normal px-2" t-if="not user['website_published']">Unpublished</span>
                 <strong class="text-muted" t-esc="user['rank']"/>
                 <div class="h3 my-2" t-if="user['karma_gain']">
-                    <span class="badge badge-pill badge-success px-3 py-2" >+ <t t-esc="user['karma_gain']"/> XP</span>
+                    <span class="badge badge-pill badge-success px-3 py-2" ><t t-if="user['karma_gain'] > 0">+ </t><t t-esc="user['karma_gain']"/> XP</span>
```

Comment:
```suggestion
                    <span class="badge badge-pill badge-success px-3 py-2" t-if="user['karma_gain'] > 0">+ <t t-esc="user['karma_gain']"/> XP</span>
                    <span class="badge badge-pill badge-warning px-3 py-2" t-if="user['karma_gain'] < 0"><t t-esc="user['karma_gain']"/> XP</span>
```
I think gaining negative karma it's not a "success". So instead should be yellow the badge. Red would be with badge-danger, as you prefer.

Reply:
thanks for the review and suggestion @MiquelRForgeFlow .
i am not sure whether i can make change such a change in stable.

@tde-banana-odoo is fine to apply this?
thanks

Reply:
@tde-banana-odoo if you have time, looking for your feedback on this

Reply:
You could use logic inside the badge (moving the `badge-success` to a `t-attf-class` attribute), which would likely be harmless in stable without requiring an additional t-if. 

```xml
<span class="badge badge-pill px-3 py-2" t-attf-class="'badge-success' if user['karma_gain']> 0 else ''">...</span>
```
I suppose something similar could be tried for the plus sign (in the `t-esc`).

Reply:
hi @flch-odoo ,
updated as per the suggestion.

positive gain:
![Screenshot from 2023-02-14 20-53-23](https://user-images.githubusercontent.com/27989791/218818740-2a82539f-3586-457d-8e6e-1aa1d73257b0.png)



negative gain:
![Screenshot from 2023-02-14 20-53-47](https://user-images.githubusercontent.com/27989791/218818732-d292267e-1051-44cc-91a3-dd111d566e8d.png)

will this be okay

Reply:
ping @flch-odoo